Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalTBModuleX.cc

```diff
@@ -209,17 +207,16 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
 #endif
   const auto& materials = args.value<std::vector<std::string> >("MaterialNames");  // Material names in each layer
   const auto& names = args.value<std::vector<std::string> >("VolumeNames");        // Names of each layer
-  const auto& layerThick = args.value<std::vector<double> >("Thickness");          // Thickness of the material
+  auto layerThick = args.value<std::vector<double> >("Thickness");                 // Thickness of the material
```

Comment:
why is this now a copy rather than const ref? It doesn't seem like the values are changed anywhere later.

Reply:
put back